lib/fluent/plugin/kinesis_helper/client.rb

```diff
@@ -38,6 +38,8 @@ module ClientParams
           config_param :duration_seconds, :integer, default: nil
           desc "A unique identifier that is used by third parties when assuming roles in their customers' accounts."
           config_param :external_id, :string, default: nil, secret: true
+          desc "A http proxy url for requests to aws sts service"
+          config_param :sts_http_proxy, :string, default: nil
```

Comment:
Could you make this parameter secret like this:

```ruby
config_param :sts_http_proxy, :string, default: nil, secret: true
```

Since this parameter can contain username and password in plain text.

Reply:
@riywo, your request for change makes total sense. I will push the corresponding change.
Also, I will idd license statement to the PR description.